js/ui/autocomplete.js

```diff
@@ -123,7 +123,9 @@ const Autocomplete = DropDownList.inherit({
 
     _renderDimensions: function() {
         this.callBase();
-        this._dimensionChanged();
+
+        this._updatePopupWidth();
+        this._popup && this._updatePopupDimensions();
```

Comment:
Let's rename `_updatePopupDimensions` with `_updateListDimensions` everywhere? It seems to be more suitable for me. Moreover, logically popup width update is a part of popup dimensions update, so actual code looks strange

Reply:
https://github.com/DevExpress/DevExtreme/pull/23483/commits/709cfa030c245fe825d90854ffcbc9655bae3ca2